test/spec/relative-date-filter.js

```diff
@@ -36,7 +36,7 @@ describe('Filter: relativeDate', function() {
     expect(relativeDate('2013-09-06T12:00:00Z')).toEqual('yesterday')
     expect(relativeDate('2013-09-06T00:00:00Z')).toEqual('yesterday')
     expect(relativeDate('2013-09-05T22:59:59Z')).toEqual('2 days ago')
-    expect(relativeDate('2013-09-01')).toEqual('6 days ago')
+    expect(relativeDate('2013-09-01T12:00:00Z')).toEqual('6 days ago')
```

Comment:
This worries me a little -- is this because of the `getDate` -> `getUTCDate` change?